bndtools.core/src/bndtools/views/repository/RepositoriesView.java

```diff
@@ -443,6 +460,31 @@ public IStatus runInWorkspace(IProgressMonitor monitor) throws CoreException {
         }.schedule();
     }
 
+    private void configureOfflineAction() {
+        Workspace workspace = Central.getWorkspaceIfPresent();
```

Comment:
There is a call to `workspace.setOffline(value);` just after creating the `Workspace` object in `getWorkspace()`. Is that too late?